### ISSUE TITLE
Support multiple drivers when generating migrations

### ISSUE THREAD

**Issue by koox00** [go-morph/morph#40]
More of a proposition, 
since `morph` supports multiple drivers, it would be nice to be able to
generate migration files for multiple drivers in one command.

e.g:
```bash
morph generate create_users --drivers mysql,postgres --dir ./foo
```

**Comment by agnivade** [comment]
Just a nit on the flag design: it might be cleaner to have multiple flag instances like `--driver mysql --driver postgres` instead. @isacikgoz - WDYT?

**Comment by isacikgoz** [comment]
Hmm.. Rather than having conflicting flags, we should only use one flag. It can be either single or multiple. Both works for me.

**Comment by agnivade** [comment]
Can you clarify what you mean by conflicting flags?

**Comment by isacikgoz** [comment]
I mean `--driver` and `--drivers` would essentially be the same flag. I think we should choose only one.

**Comment by agnivade** [comment]
Oh I totally missed that the flag name was different. Yeah I'd go with multiple instances of `--driver` just to avoid a breaking change with the flag name. 0/5 though.

**Comment by koox00** [comment]
> I mean `--driver` and `--drivers` would essentially be the same flag. I think we should choose only one.

yes yes, I wasn't thinking of changing the flag to `--drivers`, consider it a typo
